File: LV2/숫자의 표현/solution.py
def solution(n):
    answer = 0
    i = 1
    while i<=n :
        tmp = 0
        j = i
        while j<=n :
            tmp+=j
            if tmp == n:
                answer+=1
                break
            elif tmp>n:
                break
            j+=1
        i+=1
    return answer


# https://school.programmers.co.kr/learn/courses/30/lessons/12924?language=python3

File: LV2/숫자의 표현/solution.js
function solution(n) {
    var answer = 1;
    for(var i = 1; i<=n/2; i++){
        var tmp = 0;
        for(var j = i; j<=n; j++){
            tmp+=j;
            if(tmp==n){
                answer++;
                break;
            }else if(tmp>n){
                break;
            }
        }
    }
    return answer;
}

//https://school.programmers.co.kr/learn/courses/30/lessons/12924